src/modules/project.js

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable import/no-cycle */
+import { addToLocalStorage, addSelectedProject } from './localStorage';
+/* eslint-enable import/no-cycle */
+
+const projectInput = document.getElementById('new_project_input');
+const projectForm = document.getElementById('project_form');
+const projectList = document.getElementById('data-projects');
+const defaultProject = document.getElementById('data-default-project');
+const taskTitle = document.getElementById('task_title');
+const taskCounter = document.getElementById('task_count');
+const taskList = document.getElementById('task_list');
+const taskDiv = document.getElementById('task_body');
+const taskContainer = document.getElementById('tasks');
+const taskForm = document.getElementById('task_form');
+const addTaskBtn = document.getElementById('new_task_button');
+const cancelTaskBtn = document.getElementById('cancel_task_button');
+const taskInput = document.getElementById('new_task_input');
+const dueDate = document.getElementById('date');
+const priority = document.getElementById('priority_dropdown');
+
+let listOfProjects = [];
+
+let selectedProjectId;
+
+// The default project is active when clicked
+defaultProject.addEventListener('click', (e) => {
+  e.target.classList.add('active');
+  taskTitle.textContent = 'Default Project';
+
+  taskCounter.textContent = null;
+});
+
+// Create a project
+const projectItem = (item) => ({ id: Date.now().toString(), item, tasks: [] });
+
+// Create project item/task
+const taskItem = (item) => ({
+  id: Date.now().toString(), item, date: dueDate.value, priority: priority.value, complete: false,
+});
+
+// Task form
+taskForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  e.target.style.display = 'none';
+  addTaskBtn.style.display = 'block';
```

Comment:
Here for example you can create a CSS Class called .block and .none in your style.css then you can add to addTaskBtn